### ISSUE TITLE
Now drei’s OrbitControls works on React Native again?

### ISSUE THREAD

**Issue by tomsoderlund** [TiagoCavalcante/r3f-native-orbitcontrols#49]
Seems it's working now?

    import React from 'react'
    import { Canvas } from '@react-three/fiber/native'
    import { OrbitControls } from '@react-three/drei/native'

    const Plane: React.FC = () => {
      return (
        <mesh rotation={[0, 0, 0]} position={[0, 0, 0]}>
          <planeGeometry args={[1.8, 3.2]} />
          <meshStandardMaterial color='lightblue' />
        </mesh>
      )
    }

    const App3D: React.FC = () => {
      return (
        <Canvas>
          <OrbitControls />
          <ambientLight intensity={1} />
          <pointLight position={[0, 0, 1]} intensity={1.5} />
          <Plane />
        </Canvas>
      )
    }
    export default App3D

**Comment by frossi85** [comment]
Hi @tomsoderlund, would you mind telling me all the library versions you use, since I cannot make it work? Please include the Expo/React Native version as well. I am more than sure that a lot of people will find it very useful

**Comment by TiagoCavalcante** [comment]
Yes, drei's OrbitControl is working on React Native now. The relevant PR is the following: https://github.com/pmndrs/react-three-fiber/pull/3252 

For me it worked with @react-three/drei 10.7.7, @react-three/fiber 9.6.1 and expo-gl 16.0.10.

However the following issue still seems to be present: https://github.com/pmndrs/native/issues/3
